ebu_tt_live/bindings/converters/ebutt3_ebuttd.py

```diff
@@ -273,18 +273,11 @@ def convert_div(self, div_in, dataset):
         return new_elem
 
     def convert_p(self, p_in, dataset):
-        if p_in.pushed_computed_begin_time:
-            p_in.begin = self._process_timing_from_timedelta(p_in.pushed_computed_begin_time)
-        if p_in.pushed_computed_end_time:
-            p_in.end = self._process_timing_from_timedelta(p_in.pushed_computed_end_time)
-        if p_in.is_timed_leaf() == False:
-            p_in.begin = None
-            p_in.end = None
         new_elem = d_p_type(
             *self.convert_children(p_in, dataset),
             space=p_in.space,
-            begin=self._process_timing_type(p_in.begin, dataset=dataset),
-            end=self._process_timing_type(p_in.end, dataset=dataset),
+            begin=self._process_timing_from_timedelta(p_in.computed_begin_time) if p_in.is_timed_leaf() else None  ,
```

Comment:
Code style nit: Strange additional whitepace before the `,` at the end of the line.